sdk/batch/Microsoft.Azure.Batch.FileStaging/src/Microsoft.Azure.Batch.FileStaging.csproj

```diff
@@ -17,19 +17,25 @@
     </PackageReleaseNotes>
   </PropertyGroup>
   <PropertyGroup>
-    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
   </PropertyGroup>
-
   <ItemGroup>
-    <PackageReference Include="System.Text.Encodings.Web" />
-    <PackageReference Include="WindowsAzure.Storage" />
+    <PackageReference Include="System.Text.Encodings.Web" VersionOverride="4.7.2" />
```

Comment:
Any reason you need to override this version?

Reply:
I believe it was to satisfy multi-level dependencies between the packages. Azure.Storage ultimately requires at least version 4.7.2 of System.Text.Encodings.Web.